app/models/speaker.rb

```diff
@@ -75,6 +75,10 @@ def gravatar_hash
 #  info               :text
 #  created_at         :datetime
 #  updated_at         :datetime
+#  age_range          :string
+#  ethnicity          :string
+#  first_time_speaker :boolean
+#  pronouns           :string
```

Comment:
These documentation changes seem out of scope for this PR, no?

Reply:
@adarsh normally the annotate models task gets run automatically but somehow this must have not been run in a recent migration so it did not get committed. This is going to keep showing up so we should either just accept it as part of this PR or I can make a separate tiny PR for it.